packages/esm-patient-chart-app/src/visit/hooks/useDefaultVisitLocation.tsx

```diff
@@ -0,0 +1,24 @@
+import { type FetchResponse, type Location, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import { useEffect } from 'react';
+import useSWR from 'swr';
+
+/**
+ * Fetches the default visit location based on the location and the restrictByVisitLocationTag
+ * If restrictByVisitLocationTag is true, it fetches the nearest ancestor location that supports visits, otherwise it returns the passed-in location
+ *
+ * @param location
+ * @param restrictByVisitLocationTag
+ */
+export function useDefaultVisitLocation(location: Location, restrictByVisitLocationTag: boolean) {
+  let url = `${restBaseUrl}/emrapi/locationThatSupportsVisits?location=${location?.uuid}`;
```

Comment:
I think we need to check the feature flag for whether emrapi is installed

Reply:
Good point... I was going this would be "buyer beware" and the implementer would need to know that the EMR API module needed to be installed, but no reason *not* to add this check as well.